.github/workflows/luatest.yml

```diff
@@ -0,0 +1,79 @@
+name: luatest
```

Comment:
We should incorporate luatest into our building/testing pipelines, including packaging. It should be easy for pure build+test, but packaging requires that everything is in the source package. So, maybe, enable luatest as submodule of test-run?